packages/connection/src/class-manager.php

```diff
@@ -69,6 +69,13 @@ public static function configure() {
 		if ( ! wp_next_scheduled( 'jetpack_clean_nonces' ) ) {
 			wp_schedule_event( time(), 'hourly', 'jetpack_clean_nonces' );
 		}
+
+		add_filter(
+			'jetpack_constant_default_value',
```

Comment:
Yes, that's exactly what I was suggesting, thanks for implementing it that way!
One thing I'd like to suggest as well is can we set this filter in the connection's `configure` method? Just so that the filter is always set as soon as the code loads?

Reply:
Talking it through in a DM I realized that it's exactlyin the `configure` method already.